src/data.rs

```diff
@@ -64,6 +64,48 @@ pub(crate) fn generate_copyright_asset(options: &Config) -> CDResult<Vec<u8>> {
     Ok(copyright)
 }
 
+/// Compress man page assets per Debian Policy.
+/// 
+/// # References
+/// 
+/// https://www.debian.org/doc/debian-policy/ch-docs.html#manual-pages
+/// https://lintian.debian.org/tags/manpage-not-compressed.html
+pub fn compress_man_pages(options: &mut Config, listener: &dyn Listener) -> CDResult<()> {
+    let mut indices_to_remove = Vec::new();
+    let mut new_assets = Vec::new();
+
+    for (idx, asset) in options.assets.resolved.iter().enumerate() {
+        let target_path_str = asset.target_path.to_string_lossy();
+        if target_path_str.starts_with("usr/share/man/") &&
+           !target_path_str.ends_with(".gz")
+        {
+            listener.info(format!("Compressing '{}'", asset.source.path().unwrap().to_string_lossy()));
```

Comment:
You could improve that with unwrap_or(Path::new("-")).display()

Reply:
See https://github.com/mmstick/cargo-deb/pull/134.